ppx/ppx_js/lib_internal/ppx_js_internal.ml

```diff
@@ -25,12 +25,15 @@ open Parsetree
 
 let nolabel = Nolabel
 
+exception Avoid_uncaught_raising of Location.Error.t
```

Comment:
Should this be called `Syntax_error` or something ?

Reply:
Ok, I've done that.